src/backend/cuda/kernel/exampleFunction.cuh

```diff
@@ -0,0 +1,37 @@
+/*******************************************************
```

Comment:
Probably should comment on how this file will be compiled at runtime.

Reply:
I believe that will go here
https://github.com/arrayfire/arrayfire/wiki/How-to-Add-a-New-Function-to-ArrayFire